backend/src/main/java/com/redhat/cloud/notifications/routers/UserPreferencesForPolicy.java

```diff
@@ -43,6 +44,9 @@ public static class V1 extends UserPreferencesForPolicy {
     @Deprecated
     public UserConfigPreferences getPreferences(
         @Context SecurityContext sec, @PathParam("bundleName") String bundleName, @PathParam("applicationName") String applicationName) {
+        if (isServiceAccountAuthentication(sec)) {
+            throw new ForbiddenException("This api can't be used form a service account authentication");
```

Comment:
There's a typo in `from` but since it'll be fixed, I would use `with` instead 😄 
```suggestion
            throw new ForbiddenException("This api can't be used with a service account authentication");
```